db/migrate/201703030143550_set_a_lot_of_stuff_to_not_null.rb

```diff
@@ -0,0 +1,17 @@
+class SetALotOfStuffToNotNull < ActiveRecord::Migration[5.0]
+  def change
```

Comment:
we should just change this in the character migration directly, because we don't have any data that will break from just changing it there :)